### PR TITLE
Clean up on rate_diags_calc and set_rates subroutines

### PR DESCRIPTION
**Pull request by singhbalwinder**
These are fairly small routines. I did some cleanup, added units, and removed unused variables.

### CODE REVIEW

**Comment by singhbalwinder** [comment]
I did some formatting, so please hide whitespaces while reviewing.

**Review by jeff-cohere — APPROVED** [review]

File: components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_rxt_rates_conv.F90 [hunk 1]
```diff
@@ -4,16 +4,17 @@ module mo_rxt_rates_conv
   private
   public :: set_rates
 contains
-   subroutine set_rates( rxt_rates, sol, ncol )
-      real(r8), intent(inout) :: rxt_rates(:,:,:)
-      real(r8), intent(in) :: sol(:,:,:)
-      integer, intent(in) :: ncol
-      rxt_rates(:ncol,:,     1) = rxt_rates(:ncol,:,     1)*sol(:ncol,:,     2)                                                ! rate_const*H2O2
```
Comment (tangshuaiqi) [inline components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_rxt_rates_conv.F90:11]: maybe remove the excessive whitespaces as well?

**Review by tangshuaiqi — APPROVED** [review]
Just a minor comment.

File: components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_rxt_rates_conv.F90 (hunk 1 above)
Comment (singhbalwinder) [reply components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_rxt_rates_conv.F90:11]: I have removed these white spaces.

**Comment by singhbalwinder** [comment]
$ ./cs.status.comp_tmp_rate_diags_calc_mam4_org_v2_baselines -s
comp_tmp_rate_diags_calc_mam4_org_v2_baselines: 9 tests
  PASS ERP_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS ERS.ne4pg2_oQU480.F2010.compy_intel
  PASS PEM_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS PET.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P1x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P32x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_Ly1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS.ne30pg2_EC30to60E2r2.F2010.compy_intel

**Pull request merged**